src/components/components/Component.js

```diff
@@ -24,8 +27,42 @@ export default class Component extends React.Component {
     super(props);
     this.state = {
       entity: this.props.entity,
-      name: this.props.name
+      name: this.props.name,
+      nameToLinkMap: null,
+      nameList: []
     };
+    this.setObjects(this);
+  }
+
+  setObjects(self){
+    const baseUrl = process.env.REACT_APP_ADMIN_BACKEND_URL;
+    const baseEndpoint = process.env.REACT_APP_ADMIN_BASE_ENDPOINT;
+    const getUrl = baseUrl + baseEndpoint + "assets";
+
+    axios.get(getUrl, {
+        headers: {
+            "Content-Type": "application/json",
+        },
+    })
+    .catch((error) => {
+      if (error.response){
+        alert("URL: " + getUrl + "\nTitle: " + error.response.data.title + "\nMessage: " + error.response.data.message);
+      } else if (error.request){
+        alert("No response from URL: " + getUrl);
+      } else{
+        alert(error.message);
+      }
+    })
+    .then(function (response) {
+      const assets = response.data.assets;
+      let nameToLinkMap = new Map();
+      let nameList = [];
+      assets.forEach(function( item, index) {
+        nameToLinkMap[item.name] = baseUrl+"static/"+item.s3_key;
```

Comment:
Can you add the `static/` part as part of a variable to the `baseUrl` on like line 41? So we don't have to hunt for this once we switch to prod. Because prod won't need `static/` added.